src/uct/ib/rc/base/rc_iface.h

```diff
@@ -76,6 +82,16 @@ struct uct_rc_iface {
         unsigned             tx_ops_mask;
         unsigned             rx_inline;
         uint16_t             tx_moderation;
+
+        /* Threshold to send "soft" FC credit request. The peer will try to
+         * piggy-back credits grant to the counter AM, if any. */
+        uint16_t             fc_sreq_thld;
+
+        /* Threshold to sent "hard" credits request. The peer will grant
+         * credits in a separate AM as soon as it handles this request. */
+        uint16_t             fc_hreq_thld;
+
+        uint16_t             fc_wnd_size;
```

Comment:
in ud congestion avoidaince i use cwnd to be consistant with tcp stack naming convention. I suppose fc_wnd is ok.